go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -31,6 +31,11 @@ const (
 	egressFirewallAppliedCorrectly = "EgressFirewall Rules applied"
 	egressFirewallAddError         = "EgressFirewall Rules not correctly added"
 	aclDeleteBatchSize             = 1000
+	// transaction time to delete 80K ACLs from 2 port groups is ~4.5 sec.
+	// transaction time to delete 80K acls from one port group and add them to another
+	// is ~3 sec
+	// Therefore this limit is safe enough to not exceed 10 sec transaction timeout.
+	aclChangePGBatchSize = 80000
```

Comment:
another note to self: I think we need to do something with libovsdb to take the encoding time out of the transaction time.